client/components/NewMessage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import people from '../people.js'
+
+const styles = {
+  borderRadius: '150px',
```

Comment:
That makes sense. I actually didn't pay much attention to this: I copy and pasted from the other component which has this styling